edge/pkg/edged/edged.go

```diff
@@ -213,6 +213,7 @@ type edged struct {
 	rootDirectory      string
 	gpuPluginEnabled   bool
 	version            string
+	labels             map[string]string
```

Comment:
What's this labels used?

Reply:
Because I don't want to get labels form config yaml again,so I make a field of labels in edged struct.

Reply:
Did not find where this label was used..

Reply:
```initialNode()``` this function

Reply:
Where's e.label used after init in initialNode()?

Reply:
sorry, this code be omitted and I have added

Reply:
Sounds good.